wemake_python_styleguide/config.py

```diff
@@ -0,0 +1,27 @@
+# -*- coding: utf-8 -*-
+
+import configparser
+from pathlib import Path
+from typing import Optional
+
+BASE_DIR = Path(__file__).resolve().parent.parent
```

Comment:
Sorry, but that's not how it is supposed to work.
You can have different setups and `flake8` has API to get the job done.

See, how it is implemented in a similar project: https://github.com/zheller/flake8-quotes/blob/master/flake8_quotes/__init__.py#L78-L146